src/fixed.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2013-2017, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ffi;
+use glib::translate::*;
+
+use Container;
+use ContainerExt;
+use Fixed;
+use IsA;
+use Value;
+use Widget;
+
+fn has_widget<O: IsA<Fixed> + IsA<Container>, T: IsA<Widget>>(c: &O, item: &T) -> bool {
+    skip_assert_initialized!();
+    let t: *mut ffi::GtkContainer = c.to_glib_none().0;
+    let t: Container = unsafe { from_glib_none(t) };
+    let widgets = t.get_children();
+    let this: *mut ffi::GtkWidget = item.to_glib_none().0;
+    widgets.iter().find(|w| {
+        let w: *mut ffi::GtkWidget = w.to_glib_none().0;
+        w == this
+    }).is_some()
```

Comment:
Please change to this to remove unneeded conversions
```
unsafe {
    let stash = c.to_glib_none();
    let t: *mut ffi::GtkContainer = stash.0;
    let glist = ffi::gtk_container_get_children(t);
    let found = !glib_ffi::g_list_find(glist, item.to_glib_none().0 as _).is_null();
    glib_ffi::g_list_free(glist);
    found
}
```

Reply:
That's the first code I wrote but I thought we might want to avoid C code as much as possible. But if you prefer as is, fine by me! :D

Reply:
Hm, missed that `stash` and `t` unneeded here and can be replaced with direct `c.to_glib_none().0`